src/Concerns/MakesAssertions.php

```diff
@@ -215,11 +215,14 @@ public function assertDontSeeLink($link)
      */
     public function seeLink($link)
     {
-        $selector = trim($this->resolver->format("a:contains('{$link}')"));
+        $selector = trim($this->resolver->format("a[href*='{$link}']"));
```

Comment:
I think it's wrong replacement. 

for example  if I call
```
$browser->seeLink('Projects')
```
`a:contains('{$link}')` Will return link with content 'Projects'

But in your implementation it is looking in `href` attribute and not in text .